db/migrate/20221214171132_create_messages.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateMessages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :messages do |t|
+      t.string :greeting
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] I am getting eslint errors when I
<img width="1440" alt="Screen Shot 2022-12-16 at 18 02 16" src="https://user-images.githubusercontent.com/44745855/208139624-e930726b-7790-4d51-900e-f0a435335576.png">
 run the server please address them

Reply:
- [ ] Make sure you are fetching greetings on the frontend